app/javascript/components/components/Header.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import { Link } from 'react-router-dom'
 import { Nav, NavItem } from 'reactstrap'
 import Home from './../pages/Home'
+import ApartmentIndex from '../pages/ApartmentIndex'
```

Comment:
There shouldn't be any reason for you to need to import Home or ApartmentIndex to this component.